sites/default/menu_data.json

```diff
@@ -1,73 +1,73 @@
-[ { 
+[ {
     "label" : "Calendar",
     "menu_id" : "cal0",
     "target" : "cal",
     "url" : "/interface/main/main_info.php",
     "children" : [ ],
     "requirement" : 0
-}, { 
+}, {
     "label" : "Flow Board",
     "menu_id" : "pfb0",
     "target" : "flb",
     "url" : "/interface/patient_tracker/patient_tracker.php?skip_timeout_reset=1",
     "children" : [ ],
     "requirement" : 0,
     "global_req" : "!disable_pat_trkr"
-}, { 
+}, {
     "label" : "Messages ",
     "menu_id" : "msg0",
     "target" : "pat",
     "url" : "/interface/main/messages/messages.php?form_active=1",
     "children" : [ ],
     "requirement" : 0
-}, { 
+}, {
     "label" : "Patient/Client",
     "menu_id" : "patimg",
-    "children" : [ { 
+    "children" : [ {
         "label" : "Finder",
         "menu_id" : "fin0",
         "target" : "fin",
         "url" : "/interface/main/finder/dynamic_finder.php",
         "children" : [ ],
         "requirement" : 0
-    }, { 
+    }, {
         "label" : "Add New/Search",
         "menu_id" : "new0",
         "target" : "pat",
-        "url" : "/interface/new/new.php",
+        "url" : "/interface/new/new_comprehensive.php",
```

Comment:
i guess this was the key thing to do here approving these changes looks good to me

Reply:
Basically, you have to migrate the menu changes to any /sites directories you are using.  This is always the case for menu upgrades, which is why Terry wanted to the menu in the database in the first place.  Upgrading each site file is a pain.  That means that since we don't even allow the normal use of the /default site anymore (read only essentially), you always have to manually patch the sites directories in use.
  Menu admin could use a button that says "update from default" or something like that to grab the new menus too.  Or even a save/load function.

Reply:
I could have sworn that he had the change in the default site.  It was in all the roles stuff though.  I admit I did it manually because I was not in default.  Approving mad merging.